README.md

```diff
@@ -1,2 +1,24 @@
 # Reviews
-This will serve up the reviews (paginated), have a "more+" option to expand the review, an avatar and an aggregate stars value
+
+To setup and use this repo, please follow these steps:
+
+For Development:
+
+1) npm install
+2) Have MySQL5.7 installed and started
+3) npm run seed (will seed the MySQL db with sample data)
+4) npm run build (to start webpack bundling)
+5) npm run start-dev (starts the server)
+
+For your Proxy Server:
```

Comment:
Proxy doesn't use mysql, isn't it?

Reply:
no, but if you're going to use Reviews Service on your proxy server, you need the Mysql db with the service